modules/gitops-release-notes-1-17-3.adoc

```diff
@@ -0,0 +1,77 @@
+= Release notes for {gitops-title} 1.17.3
+
+{gitops-title} 1.17.3 is now available on {OCP} 4.12, 4.14, 4.15, 4.16, 4.17, 4.18, and 4.19.
+
+[IMPORTANT]
+====
+After completing the upgrade, run the `audit-operator-roles.sh` audit script to review namespace-scoped access created by the {gitops-title} Operator. The script identifies `Role` and `RoleBinding` objects that grant cross-namespace access when the _Applications in any namespace_ feature or _ApplicationSets in any namespace_ feature is enabled using the Argo CD custom resource (CR) (`.spec.sourceNamespaces` and `.spec.applicationSet.sourceNamespaces`). These features are disabled by default. The script scans all namespaces and reports `Role`/`RoleBinding` combinations that meet the following conditions:
+
+* Grant access to the `argoproj.io/Application` resource
+* Are labeled with `app.kubernetes.io/part-of=argocd`
+* Bind to a service account in a different namespace, resulting in cross-namespace access
+
+Review the script output to ensure that cross-namespace access is limited to the intended namespaces only.
+
+For more information about the script, see link:https://github.com/redhat-developer/gitops-operator/blob/master/scripts/audit-namespace-roles/audit-operator-roles.sh[Audit Operator script].
+
+For more information about the steps to execute the script, see link:https://github.com/redhat-developer/gitops-operator/blob/master/scripts/audit-namespace-roles/README.md[Audit Operator script readme].
+====
+
+[id="errata-updates-1-17-3_{context}"]
+== Errata updates
+
+[id="RHSA-2025:23206-gitops-1-17-3-security-update-advisory_{context}"]
+=== RHSA-2025:23206 - {gitops-title} 1.17.3 security update advisory
+
+Issued: 2025-12-15
+
+The list of security fixes included in this release is available in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2025:23206[RHSA-2025:23206]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-17-3_{context}"]
+== Fixed issues
+
+git-lfs dependency update::
+* With this update, the `git-lfs` dependency in the Argo CD container image is updated to `v3.7.1`.
++
+link:https://issues.redhat.com/browse/GITOPS-8142[GITOPS-8142]
+
+Redis version update::
+* With this update, the Redis version included with {gitops-title} is upgraded to `v7.2.11`.
++
+link:https://issues.redhat.com/browse/GITOPS-8019[GITOPS-8019]
+
+GitOps Operator pulls images from registry.redhat.io::
+Before this update, when the `.spec.version` field was set in the Argo CD CR, the {gitops-title} Operator incorrectly pulled upstream images from `quay.io`. With this update, this issue is fixed. The {gitops-title} Operator pulls images from `registry.redhat.io` and only uses certified images from the Red Hat registry.
```

Comment:
```suggestion
Before this update, when the `.spec.version` field was set in the Argo CD CR, the {gitops-title} Operator incorrectly pulled upstream images from `quay.io`. With this release, this issue is fixed. The {gitops-title} Operator pulls images from `registry.redhat.io` and only uses certified images from the Red Hat registry.
```
Consider updating sentences following the style guide: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-fixed-issues

Consider doing this for all the fixed issues release notes mentioned below.